platform/darwin/src/MGLConversion.h

```diff
@@ -68,8 +68,15 @@ inline OptionalNSObjectValue objectMember(const id value, const char *key) {
     }
 }
 
-// Not implemented (unneeded for MGLStyleFunction conversion):
-// optional<Error> eachMember(const NSObject*, Fn&&)
+inline optional<Error> eachMember(const id value, std::function<optional<Error>(const std::string&, const id)> Fn) {
+    NSCAssert([value isKindOfClass:[NSDictionary class]], @"Value must be an NSDictionary for get(string).");
```

Comment:
This message will need to be reworded once #8074 is implemented, as `get(string)` won’t be the syntax for obtaining an attribute value from a feature.